diff --git a/drush_make.project.inc b/drush_make.project.inc
index 76b71b2..497d835 100644
--- a/drush_make.project.inc
+++ b/drush_make.project.inc
@@ -392,3 +392,10 @@ class DrushMakeProject_Translation extends DrushMakeProject {
     }
   }
 }
+
+class DrushMakeProject_Sites extends DrushMakeProject {
+  protected function __construct(&$project) {
+    parent::__construct($project);
+    $this->contrib_destination = 'sites';
+  }
+}
